.circleci/config.yml

```diff
@@ -579,6 +586,53 @@ jobs:
           name: Build SPM Core integration test
           command: cd Tests/SPMTest/CoreSPMTest && xcodebuild -scheme CoreSPMTest -destination "platform=iOS Simulator,OS=15.0,name=iPhone 13 Pro Max" build
       
+  generate-docs-job:
+    parameters:
+      xcode:
+        type: string
+        default: "12.4.0"
+    macos:
+      xcode: << parameters.xcode >>
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - checkout
+      - *install-cocoapods-installation
+      - *prepare-netrc-file
+      - *install-mbx-ci
+      - *restore-cache-podmaster
+      - *restore-cache-cocoapods-installation
+      - run:
+          name: Generate docs
+          command: |
+            OUTPUT=$( echo << pipeline.git.tag >> | sed 's/^v//' )
+            OUTPUT="${OUTPUT:-documentation}"
+            echo "export OUTPUT=$OUTPUT" >> $BASH_ENV
+            OUTPUT="${OUTPUT}" scripts/document.sh 2>&1 | tee docs.output && ! egrep -e "(WARNING)|(USR)" docs.output
```

Comment:
#2078 requests checking for missing USR warnings. Some of the warnings are expected due to `#if` usage, but others are not. How should we distinguish the two cases?

Reply:
Is there a pattern for the expected warnings? There are no warnings at this time.